lib/connect-backbone-to-react.js

```diff
@@ -124,11 +130,25 @@ module.exports = function connectBackboneToReact(
         // Bind event listeners for each model that changed.
         Object.keys(this.models).forEach(mapKey => {
           const model = this.models[mapKey];
-          if ((this.props.models && this.props.models[mapKey] === this.models[mapKey]) ||
-            (this.context.models && this.context.models[mapKey] === this.models[mapKey])) {
+          if ((this.props.models && this.props.models[mapKey] === model) ||
+            (this.context.models && this.context.models[mapKey] === model)) {
             return; // Did not change.
           }
 
+          // Do not attempt to create event listeners on an undefined model.
+          if (!model) {
```

Comment:
Should this block come before the first if block in this function?

Also we're looking up the propModel and the contextModel twice. Every time you try and access a property on an object you have a small hit to performance. Putting aside the micro-optimization, it may make this code easier to read if we cache that lookup at the top of the block so we don't have to repeat the boilerplate code in two locations.

Reply:
Yup, cleaned this up a bit 🙂